source/docs/software/commandbased/commands-v3/getting-started.rst

```diff
@@ -0,0 +1,283 @@
+# Commands v3: Imperative Commands with Coroutines (Advanced)
```

Comment:
It seems odd that an article labeled advanced is also called getting started, and second in the article list. 

https://www.youtube.com/watch%3Fv%3DrsRjQDrDnY8